account-management-app/src/main/java/org/duracloud/account/app/controller/AbstractController.java

```diff
@@ -136,4 +176,32 @@ protected void setFailureFeedback(String message, RedirectAttributes redirectAtt
         UserFeedbackUtil.addFailureFlash(message, redirectAttributes);
     }
 
+    protected boolean validateRecaptcha(final String recaptchaResponse) throws IOException {
```

Comment:
This method validates the recaptcha response generated by the client side recaptcha js library.
A call is made out to recaptcha service with the response and the account secret.